app/services/submission/SubscriptionBlockBuilder.scala

```diff
@@ -102,6 +103,18 @@ class SubscriptionBlockBuilder @Inject()(registrationMongoRepository: VatSchemeR
           case (false, _) => None
           case _ => throw new InternalServerException("[SubscriptionBlockBuilder] FRS scheme data missing when joinFrs is true")
         }
+      }),
+      optional("takingOver" -> eligibilityData.togcBlock.map { togcData =>
+        jsonObject(
+          "prevOwnerName" -> togcData.previousBusinessName,
+          "prevOwnerVATNumber" -> togcData.vatRegistrationNumber,
+          "keepPrevOwnerVATNo" -> togcData.wantToKeepVatNumber,
+          "acceptTsAndCsForTOGCOrCOLE" -> (if (!togcData.wantToKeepVatNumber) {
```

Comment:
This would be clearer if the logic was flipped - positive first is almost always more legible